packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -16,21 +15,23 @@ class AlertBox extends React.Component {
   }
 
   componentDidUpdate() {
-    if (this.props.isVisible && this.props.scrollOnShow) {
-      this.scrollToAlert();
-    }
+    this.scrollToAlert();
   }
 
-  scrollToAlert() {
+  scrollToAlert = () => {
     const isInView = window.scrollY <= this._ref.offsetTop;
 
+    if (!this.props.isVisible || !this.props.scrollOnShow) {
```

Comment:
I see, so we escape early in `scrollToAlert` so we don't need to have the conditional logic in `componentDidMount` and `componentDidUpdate`, smart! 😎